src/nvs.rs

```diff
@@ -352,6 +352,270 @@ impl<T: NvsPartitionId> EspNvs<T> {
 
         Ok(true)
     }
+
+    pub fn len_str(&self, name: &str) -> Result<Option<usize>, EspError> {
```

Comment:
Maybe `str_len`? But then also see my comments on whether we should surrender to `alloc` for this API.

Reply:
I think keeping the function makes sense independent of the alloc topic